drivers/vultr/vultr.go

```diff
@@ -0,0 +1,362 @@
+package vultr
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"time"
+
+	"github.com/ChimeraCoder/tokenbucket"
+	vultr "github.com/JamesClonk/vultr/lib"
+	"github.com/codegangsta/cli"
+	"github.com/docker/machine/drivers"
+	"github.com/docker/machine/log"
+	"github.com/docker/machine/provider"
+	"github.com/docker/machine/ssh"
+	"github.com/docker/machine/state"
+)
+
+type Driver struct {
+	APIKey            string
+	MachineID         string
+	MachineName       string
+	PublicIP          string
+	PrivateIP         string
```

Comment:
I misunderstood `PrivateNetworking` :)... So, can you _either_ add `IPAddress` which is in sync with `PublicIP`, or can you rename `PublicIP` to `IPAddress`?